src/components/Datatable/ControlModule/ControlModule.stories.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+import { MemoryRouter } from 'react-router-dom';
+
+import { Container } from '../../layout';
+import ControlModule from './ControlModule';
+import { BatchModule } from '../BatchModule';
+import { SearchBar } from '../../forms/SearchBar';
+import { ToolsTabItem } from '../ToolsTabItem';
+import { ControlModuleProps } from './ControlModule.types';
+import { mockControlModuleProps } from '../mocks/controlModule';
+
+export default {
+  title: 'components/Datatable/components/ControlModule',
+  component: ControlModule,
+  decorators: [(storyFn) => <MemoryRouter>{storyFn()}</MemoryRouter>],
+  subcomponents: { BatchModule, ToolsTabItem, SearchBar },
+} as Meta;
+
+export const playground: Story<ControlModuleProps<D>> = (args) => (
```

Comment:
Instead of D use either `Data` from  https://github.com/securityscorecard/design-system/blob/alpha/src/components/Datatable/Datatable.stories.tsx#L35 or `Record<'string', unknown>`